tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
@snowcrshd, 
> Shouldn't the mock-args parameters be surrounded by quotes, since they are separated by spaces?

I refer to this ^

This test doesn't check that it works with more than one "mock-arg" although it looks like it does.

But, yeah, I would also change the names to '--foo' '--bar'